expat/CMakeLists.txt

```diff
@@ -669,6 +669,52 @@ if(EXPAT_BUILD_FUZZERS)
                 TARGET ${target_name} PROPERTY RUNTIME_OUTPUT_DIRECTORY fuzz)
         endforeach()
     endforeach()
+
+    find_package(Protobuf REQUIRED)
+
+    # Only include libprotobuf-mutator here so we don't build it in non-fuzz
+    # configurations.
+    include(ExternalProject)
+
+    set(ProtobufMutator_PREFIX libprotobuf-mutator)
+    set(ProtobufMutator_PATH ${CMAKE_CURRENT_BINARY_DIR}/${ProtobufMutator_PREFIX}/src/${ProtobufMutator_PREFIX})
+    set(ProtobufMutator_BUILD_PATH ${ProtobufMutator_PATH}-build)
+    set(ProtobufMutator_INCLUDE_DIR ${ProtobufMutator_PATH})
+    set(ProtobufMutator_LIBRARIES ${ProtobufMutator_BUILD_PATH}/src/libfuzzer/libprotobuf-mutator-libfuzzer.a ${ProtobufMutator_BUILD_PATH}/src/libprotobuf-mutator.a)
+
+    ExternalProject_Add(
+        ${ProtobufMutator_PREFIX}
+        PREFIX ${ProtobufMutator_PREFIX}
+        GIT_REPOSITORY https://github.com/google/libprotobuf-mutator.git
+        GIT_TAG master
```

Comment:
Adding a master branch — a moving target — will be a problem to CI stability.
The latest release of [libprotobuf-mutator](https://github.com/google/libprotobuf-mutator) is of 2022-11-13 while its `master` has commits as recent as 2022-05-20. My vote for making a new release in libprotobuf-mutator so that we have something robust and non-moving for potential use in a CI. Thank you.

Reply:
I've updated the branch to build protobuf from source and pin the versions of both protobuf and libprotobuf-mutator.

Reply:
@c01db33f thanks for the pin, but there is more to it: Which release we're pinning precisely, how old the release is, and how well the project is maintained. Pinning a release of 2 years ago of a project that has stopped doing releases _would_ not be healthy. So only a recent release in [libprotobuf-mutator](https://github.com/google/libprotobuf-mutator) will really be able to show the project is healthy enough and understands need for releases, to be a dependable in 2022. What do you think?

With regard to core protobuf, I would like to understand your choice of [v3.20.1](https://github.com/protocolbuffers/protobuf/releases/tag/v3.20.1) over [v3.21.4](https://github.com/protocolbuffers/protobuf/releases/tag/v3.21.4) and [v21.4](https://github.com/protocolbuffers/protobuf/releases/tag/v21.4). Could you elaborate?